src/WebJobs.Script/Binding/CoreExtensionsScriptBindingProvider.cs

```diff
@@ -66,6 +66,7 @@ public override Collection<Attribute> GetAttributes()
 
                 string schedule = Context.GetMetadataValue<string>("schedule");
                 bool runOnStartup = Context.GetMetadataValue<bool>("runOnStartup");
+                bool useMonitor = Context.GetMetadataValue<bool>("useMonitor", true);
```

Comment:
Ah, so many `function.json`s :D I meant this file (https://github.com/Azure/azure-webjobs-sdk-script/blob/dev/schemas/json/function.json) but the other could use updating as well